sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -526,6 +526,83 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
     });
   });
 
+  it("Should skip the awards from past SFAS full time cancelled application(s) for the same student and program year when calculating the program year award totals.", async () => {
+    // Arrange
+
+    // Create the student and program year to be shared across the applications.
+    const student = await saveFakeStudent(db.dataSource);
+    // Get the program year for the start date.
+    const programYear = await db.programYear.findOne({ where: { id: 2 } });
+
+    // Current application having the first assessment in progress.
+    const currentApplication = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      { student },
+      {
+        offeringIntensity: OfferingIntensity.partTime,
+        applicationStatus: ApplicationStatus.InProgress,
+        currentAssessmentInitialValues: {
+          assessmentWorkflowId: "some fake id",
+          studentAssessmentStatus: StudentAssessmentStatus.InProgress,
+          assessmentDate: addDays(30, programYear.startDate),
+        },
+      },
+    );
+    const currentApplicationAssessmentDate =
+      currentApplication.currentAssessment.assessmentDate;
+    // The start date for the SFAS full time application record is set to the date before the first assessment date of the current application.
+    const firstLegacyApplicationStartDate = faker.date.between(
+      programYear.startDate,
+      addDays(-1, currentApplicationAssessmentDate),
+    );
+    const firstLegacyApplicationEndDate = addDays(
+      30,
+      currentApplicationAssessmentDate,
+    );
+    await db.application.save(currentApplication);
+
+    // SFAS Individual.
+    const sfasIndividual = await saveFakeSFASIndividual(db.dataSource, {
+      initialValues: {
+        lastName: student.user.lastName,
+        birthDate: student.birthDate,
+        sin: student.sinValidation.sin,
+      },
+    });
+    // Past SFAS application with the start date before the first assessment date of the current application.
+    const pastFakeSFASApplication = createFakeSFASApplication(
```

Comment:
As a suggestion, we can have a second application exactly like this one, non-cancelled, and with different awards values, to ensure only the values from this last application are returned.